homework.js

```diff
@@ -0,0 +1,124 @@
+//Q1 ->work
+const summation = function (number) {
+    let sum = 0;
+    for (let i = 0; i <= number; i++) {
+        sum = sum + i;
+    }
+    console.log(sum);
+}
+summation(5);
+console.log("------");
+//Q2 ->work
+const summationEven = function (number) {
+    let sum = 0;
+    for (let i = 0; i <= number; i++) {
```

Comment:
you can also increase the increment by 2 

`for (let i = 2; i <= n; i += 2){
        sum +=i;
    }`